pkg/apiclient/info/info.proto

```diff
@@ -42,4 +49,10 @@ service InfoService {
   rpc GetUserInfo(GetUserInfoRequest) returns (GetUserInfoResponse) {
     option (google.api.http).get = "/api/v1/userinfo";
   }
+  rpc CollectEvent(CollectEventRequest) returns (CollectEventResponse) {
+    option (google.api.http) = {
+      post : "/api/v1/tracking/event"
+      body : "*"
```

Comment:
try changing ithis to `body: "`param`"`

Reply:
I tried it, but it didn't work.
After changing it and generating code from this proto file, I met the error like 
```sh
--grpc-gateway_out: repeated field not allowed in field path: param in param
```
It seems to be related to https://github.com/grpc-ecosystem/grpc-gateway/issues/765 and unfortunately it's still not supported.

Reply:
Hmm, my smart idea of using `map<>` is look less smart by the moment.

Lets pivot, go back to your original idea. Lets just have a singe `name` field. We can add other in the future if we want them.

I hope you don't mind my about face.

Reply:
I understand! Thank you very much for your advice :)